Content.Client/GameObjects/EntitySystems/GasTileOverlaySystem.cs

```diff
@@ -82,13 +90,84 @@ public override void Initialize()
                 _fireFrameDelays[i] = state.GetDelays();
                 _fireFrameCounter[i] = 0;
             }
+            
+            var overlayManager = IoCManager.Resolve<IOverlayManager>();
+            if(!overlayManager.HasOverlay(nameof(GasTileOverlay)))
+                overlayManager.AddOverlay(new GasTileOverlay());
+        }
+
+        private void HandleGasOverlayMessage(GasOverlayMessage message)
+        {
+            if (!_tileData.TryGetValue(message.GridId, out var chunks))
+            {
+                chunks = new Dictionary<MapIndices, GasOverlayChunk>();
+                _tileData[message.GridId] = chunks;
+            }
```

Comment:
This block isn't necessary because `GetOrCreateChunk` has the same code, I believe.